docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -7,17 +7,46 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-This document describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC and Mutual TLS authentication as well as the proactive authentication.
+The following section describes the Quarkus built-in authentication mechanisms for HTTP-based BASIC, FORM, and both Mutual TLS and proactive authentication.
```

Comment:
The change from `and Mutual TLS authentication as well as the ` to `and both Mutual TLS and proactive authentication` is not quite correct, `proactive authentication` is not a built in mechanism but is about controlling at what point these built in and other authentication mechanisms start working. So I think it needs to be reverted. 
Perhaps we'll move the proactive authentication section to its own doc eventually

Reply:
Ok, putting it back and removing that proactive mention.